### ISSUE TITLE
Dejavu -> Prop Courrier Sans: add details

### ISSUE THREAD

**Issue by raphaelbastide** [celiadzc/type-derivation#4]
Add the type of derivation for DejaVu to Prop Courrier Sans. I think Prop Courrier Sans is based on the metrics of DejaVu right?

Are you sure Not Courier Sans is linked to Dejavu?

**Comment by celiadzc** [comment]
Actually it's not mentionned in the FONTLOG file. But [the gif](http://strabic.fr/IMG/gif/fork-me-1413985803-1415567940.gif) say that Not Courier Sans is based on the spacing of Dejavu Sans. Prop Courier Sans takes the spacings of Dejavu Sans [(found here)](https://gitlab.com/libregraphicsmag/propcouriersans/blob/master/FONTLOG.txt).
Spacing and metrics are 2 differents things, aren't they ? Is it like this :
spacing/kerning = space between the letters 
metrics = ratio height/width (chasse)

**Comment by raphaelbastide** [comment]
What I understand is:
- The spacing of Not Courier Sans remains monospace, and inherit from Courier sans without any modification.
- The spacing of Prop Courier Sans is modified from monospace to proportional and is based on DejaVu’s.

> Not Courier Sans is based on the spacing of Dejavu Sans

Are you sure? Can you provide a screenshot?

**Comment by celiadzc** [comment]
No I'm not sure, I'm just trying to understand why in the gif it is noted this.
![fork-me-1413985803-1415567940](https://cloud.githubusercontent.com/assets/9606433/13776905/faca866c-eaac-11e5-92d2-68fc6905ee84.jpg)

**Comment by raphaelbastide** [comment]
Yeah, it is a bit blurry on this gif but I am pretty sure that:  
_Not Courier Sans_ + DejaVu’s metrics = _Prop Courier Sans_

Maybe @rlafuente can confirm?

**Comment by rlafuente** [comment]
Yes, this is mostly true! 

There are many kerning pairs and spacings that we adjusted by hand, so it is not a simple spacing transposition. Additionally, there are some characters that were edited to better fit the proportional design -- punctuation is the most evident example, and there was some trimming to the width of `i` and `l`, among other tiny tweaks.

In fact, the first version of PropCourier was not using Dejavu's spacings and was all done by hand, but later we wrote scripts to import spacings in order to make our lives easier.

**Comment by rlafuente** [comment]
Just found an [updated README](https://gitlab.com/manufacturaind/propcourier-sans) with all the necessary info.

The gif looks beautiful! And it's Manufa**c**tura btw ;-)

**Comment by raphaelbastide** [comment]
Thank you @rlafuente for those precisions and the README update!
The gif is not part of our project, I think Loraine Furter designed it for http://specimen.meteor.com/
